sky/skylet/constants.py

```diff
@@ -23,6 +23,14 @@
 SKY_REMOTE_RAY_TEMPDIR = '/tmp/ray_skypilot'
 SKY_REMOTE_RAY_VERSION = '2.9.3'
 
+# Use the same Python used for installing ray and skypilot.
```

Comment:
This deserves a longer explanation, I think. How about expanding it to include example values for each var
- SKY_RAY_CMD looks like /path/dir/.../ray
- SKY_PYTHON_CMD looks like /path/dir/.../ray
- ...

and "We do this because users may ...."

Reply:
Added. Thanks!